src/handlers/diff_header.rs

```diff
@@ -479,49 +472,48 @@ mod tests {
     #[test]
     fn test_get_file_extension_from_marker_line() {
         assert_eq!(
-            get_file_extension_from_marker_line(
-                "--- src/one.rs	2019-11-20 06:47:56.000000000 +0100"
-            ),
-            Some("rs")
+            get_filename_from_marker_line("--- src/one.rs	2019-11-20 06:47:56.000000000 +0100"),
+            Some("one.rs")
         );
     }
 
     #[test]
     fn test_get_file_extension_from_diff_header_line() {
         assert_eq!(
-            get_file_extension_from_diff_header_line_file_path("a/src/parse.rs"),
-            Some("rs")
+            get_filename_from_diff_header_line_file_path("a/src/parse.rs"),
+            Some("parse.rs")
         );
         assert_eq!(
-            get_file_extension_from_diff_header_line_file_path("b/src/pa rse.rs"),
-            Some("rs")
+            get_filename_from_diff_header_line_file_path("b/src/pa rse.rs"),
+            Some("pa rse.rs")
         );
         assert_eq!(
-            get_file_extension_from_diff_header_line_file_path("src/pa rse.rs"),
-            Some("rs")
+            get_filename_from_diff_header_line_file_path("src/pa rse.rs"),
+            Some("pa rse.rs")
         );
         assert_eq!(
-            get_file_extension_from_diff_header_line_file_path("wat hello.rs"),
-            Some("rs")
+            get_filename_from_diff_header_line_file_path("wat hello.rs"),
+            Some("wat hello.rs")
         );
         assert_eq!(
-            get_file_extension_from_diff_header_line_file_path("/dev/null"),
+            get_filename_from_diff_header_line_file_path("/dev/null"),
             None
         );
+        // TODO, handle this:
```

Comment:
Can you expand comment to clarify in what sense this is not handled?

Reply:
Oh, that is handled now :) - Left over from writing the failing tests first.